### PR TITLE
This is our first project /Embbeding images video 

### PR DESCRIPTION
**Pull request by eduardoreisalvarenga**
Please let us know what you think

### CODE REVIEW

**Comment by Ispirett** [comment]
#reviewing

File: index.html [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+        <meta charset="UTF-8">
+    </head>
+    <body>
+        <!--Top part of the web page Start -->
+        <section class="main-navbar">
+            <img src="images/ic_menu.svg" alt="Display the whole YouTube menu" id="menu-icon" width="22px" height="22px">
+            <img src="images/img_youtube_logo.svg" alt="YouTube logo" id="youtube-logo" width="80px">
+            <form action="" method="POST">
+                <input class="search-bar" type="text" name="search-bar" placeholder="Search">  
+                <div class="search-container">
```
Comment (Ispirett) [inline index.html:15]: User a button here instead of input.

**Review by Ispirett — CHANGES_REQUESTED** [review]
Hey, folks exceptional work with this project you are doing a fantastic job :1st_place_medal:  .There are just a few changes that Mircoverse requires before this project can be completed.

### Sidebar Recommended videos
- [x] **Optional** instead of img tag with images  you can use `<iframe>` with videos
- [x] **Optional** It is better to use classes instead of ids. Ids are mostly used in javascript and id's are unique so limits you from reusing them.
- [x] Please put more information in you read me about and your name.
### html errors
There are a lot of HTML errors on this page use this tool and please correct them https://validator.w3.org/

When finished with these changes please submit for another code review you can use this same pull request link.

File: index.html (hunk 1 above)
Comment (davidauza-engineer) [reply index.html:15]: Done :)

**Comment by Oghenebrume50** [comment]
Great work team everything works perfectly, submit a project completion form and move to the next project, kudos for working with Github flow. Awesome.

**Comment by davidauza-engineer** [comment]
Thank you very much guys!

**Pull request merged**